client/service/src/builder.rs

```diff
@@ -250,11 +250,15 @@ where TGen: RuntimeGenesis, TCSExt: Extension {
 		(),
 		TLightBackend<TBl>,
 	>, Error> {
+		#[cfg(not(target_os = "unknown"))]
```

Comment:
Well, the next obvious step is to adjust `config.keystore_path` to be an enum of either a path or "in memory".

Reply:
👍